test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -81,9 +79,37 @@ public void WebTestRunAllTestsWithRunSettings(RunnerInfo runnerInfo)
 
         CreateRunSettingsFile(runSettingsFilePath, runSettingsXml);
 
-        //minWebTestResultFileSizeInKB is set to 150 here as the web test has a iteration count set to 5
         //therefore, the test will run for 5 iterations resulting in web test result file size of at least 150 KB
-        WebTestRunAllTests(runnerInfo.RunnerFramework, runSettingsFilePath, 150);
+        var minWebTestResultFileSizeInKB = 150;
+        if (runnerInfo.IsNetRunner)
+        {
+            Assert.Inconclusive("WebTests tests not supported with .Netcore runner.");
+            return;
+        }
+
+        string assemblyRelativePath =
+            @"microsoft.testplatform.qtools.assets\2.0.0\contentFiles\any\any\WebTestAssets\WebTest1.webtest";
+
+        var assemblyAbsolutePath = Path.Combine(_testEnvironment.PackageDirectory, assemblyRelativePath);
+        using var resultsDirectory = TempDirectory;
+        var arguments = PrepareArguments(
+            assemblyAbsolutePath,
+            string.Empty,
+            runSettingsFilePath, FrameworkArgValue, string.Empty, resultsDirectory.Path);
+
+        InvokeVsTest(arguments);
+        ValidateSummaryStatus(1, 0, 0);
+
+        if (minWebTestResultFileSizeInKB > 0)
```

Comment:
Maybe add a comment about the case where we would have a zero size or if that's not expected maybe throw/assert.fail.

Reply:
Not my code, just moved from method that was never reused.